pkg/podvolume/defaults/defaults.go

```diff
@@ -0,0 +1,5 @@
+package defaults
```

Comment:
This package name looks too narrow to hold other codes, can we just add pkg/podvolume/configs/configs.go and put `PVCNameAnnotation` and `DefaultVolumesToFsBackup` to it?

Reply:
Sure thing